### PR TITLE
Added composer.json file as described in Issue 31

### PR DESCRIPTION
**Pull request by baldurrensch**
https://github.com/michelf/php-markdown/issues/31

### CODE REVIEW

**Comment by marijn** [comment]
:+1:

**Comment by michelf** [comment]
Looks like something that'd fit better with the new [lib branch](https://github.com/michelf/php-markdown/tree/lib) I added today. That branch should conform to PSR-0, assuming I haven't done anything wrong.

**Comment by simensen** [comment]
This composer.json will not work as it does not specify the autoloader information required to actually use the classes. I believe the following would need to be added in addition to rebasing agains the lib branch in order for the classes to be usable:

```
"autoload": {
    "psr-0": { "michelf": "" }
},
```

@michelf I'm super excited you are moving in this direction. There are a lot of people using my port ([dflydev/markdown](https://github.com/dflydev/dflydev-markdown)) and I'd be really excited to be able to reference the official php-markdown distribution as an option instead of maintaining my port indefinitely.

I'd be happy to help on any and all fronts with this! Please let me know if you have any questions on Composer, Packagist, or want help maintaining this package on Packagist once you're ready to publish it there.

**Comment by ghost** [comment]
@michelf - yes, a PSR-0 version would be better. You can always make it BC by having a `markdown.php` in the root which `require`'s the other two files.

**Comment by simensen** [comment]
@michelf How much work have you been abel to do on the lib branch? Are you close to being able to release that?

**Comment by michelf** [comment]
I haven't worked much on it. I plan to finalize, annouce, and release it at the same time the next version comes in... Which should be done in less than a month. Technically, there isn't much left to do. Perhaps only adding the composer file.

**Comment by michelf** [comment]
@drak Backward compatibility on the PHP API side is of no concern in the lib branch. It's a clean break. That's the whole point.

I still need to be able to merge with the other branches however, until they are deprecated. So this puts limits on code refactorings. But the API is a clean break.

**Comment by ghost** [comment]
You know for the php 4 version you can just add a files: require and be done with it.... then worry about the PSR-0 stuff on your `lib` branch. 

```
"autoload": {
    "files": ["markdown.php"]
}
```

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "authors": [
+        {
+            "name": "Michel Fortin",
+            "homepage": "http://michelf.com"
+        },
+        {
+            "name": "John Gruber",
+            "homepage": "http://daringfireball.net"
+        }
+    ],
```
Comment (ghost) [inline composer.json:17]: Missing compatible loader:

```
"autoload": {
    "files": ["markdown.php"]
}
```

**Comment by jason-gu** [comment]
Please add composer support, thanks

**Comment by michelf** [comment]
This has been merged on the lib branch (as discussed earlier). Note, that as this pull request was made against master, github won't close it automatically and won't show it as merged, so I have to close it manually.

**Pull request closed without merging**